zerver/tests/test_bots.py

```diff
@@ -951,3 +951,20 @@ def test_create_outgoing_webhook_bot(self, **extras):
         bot_info['payload_url'] = ujson.dumps('http://127.0.0.:5002/bots/followup')
         result = self.client_post("/json/bots", bot_info)
         self.assert_json_error(result, "Enter a valid URL.")
+
+    def test_outgoing_webhook_interface_type(self):
+        # type: () -> None
+        self.login(self.example_email('hamlet'))
+        bot_info = {
+            'full_name': 'Outgoing Webhook test bot',
+            'short_name': 'outgoingservicebot',
+            'bot_type': UserProfile.OUTGOING_WEBHOOK_BOT,
+            'payload_url': ujson.dumps('http://127.0.0.1:5002/bots/followup'),
+            'interface_type': 10,
```

Comment:
If it's likely that 10 would be used as an actual `interface_type` in the future, then can you please use another value (like -1)?